src/stories/Forms/Toggle/index.tsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import { GlobalInputProps } from 'src';
+import styled from 'styled-components';
+
+const Wrapper = styled.div`
+  height: var(--spacing-24);
+  width: 44px;
+  position: relative;
+`;
+
+const StyledInput = styled.input`
+  // the toggle button uses a checkbox for UU reasons
+  // this is then restyled to look like a toggle button
+  width: 100%;
+  height: 100%;
+  border-radius: 20px;
+  background: var(--color-grey55);
+  appearance: none;
+  outline: none;
+
+  // the before element is adding the circle to the toggle button
+  ::before {
+    height: 20px;
+    width: 20px;
+    position: absolute;
+    top: 2px;
+    left: 2px;
+    border-radius: 50%;
+    background: white;
+    transition: 0.2s ease-out;
+    content: '';
+  }
+
+  :checked {
+    background: var(--color-lavender);
+  }
+
+  :checked::before {
+    left: 22px;
+  }
+
+  :checked:hover {
+    background: var(--color-salviaBlue);
+  }
+
+  :hover {
+    background: var(--color-grey45);
+  }
+
+  :checked:disabled {
+    background: var(--color-plumbagoBlue);
+  }
+
+  :disabled {
+    background: var(--color-grey76);
+  }
+
+  :focus-visible {
+    outline-style: auto;
+    outline-color: var(--color-salviaBlue);
+    outline-offset: 3px;
+  }
+`;
+
+export interface ToggleProps extends Omit<GlobalInputProps, 'value'> {
+  className?: string;
+  value?: boolean;
+}
+
+/**
+ * Toggle component
+ */
+export const Toggle = React.forwardRef(
+  (props: ToggleProps, ref: React.RefObject<HTMLInputElement>) => {
+    const {
+      ariaDescribedBy,
+      ariaErrorMessage,
+      className,
+      hasError,
+      id,
+      isRequired,
+      name,
+      value,
+      ...additionalInputProps
+    } = props;
+    return (
+      <Wrapper className={className}>
+        <StyledInput
+          type='checkbox'
+          autoFocus
```

Comment:
Why is the autoFocus attribute being used here?
I can understand using it in dialogs and such, but to always have it enabled during page load sounds like a no go 🤔

Reply:
hehe used it to test focus color and forgot to delete it. Nice catch!